### ISSUE TITLE
Graphs section breaks responsive layout

### ISSUE THREAD

**Issue by rbarilani** [zalando-incubator/zalando.github.io-dev#16]
The graphs section breaks the layout and needs style adjustment.
@mikiobraun What do you think if in the meanwhile we move this functionality on its own branch until is ready?

**Comment by mikiobraun** [comment]
Can you be more specific what the issue is and what would be needed to fix it? I'd rather we try and fix it now. Now is as food as ever. Can't guarantee I'll have more time later.

----- Ursprüngliche Nachricht -----
Von: "Ruben Barilani" notifications@github.com
Gesendet: ‎12.‎09.‎2015 14:11
An: "zalando/zalando.github.io-dev" zalando.github.io-dev@noreply.github.com
Cc: "Mikio L. Braun" mikiobraun@gmail.com
Betreff: [zalando.github.io-dev] Graphs section breaks responsive layout (#16)

The graphs section breaks the layout and needs style adjustment.
@mikiobraun What do you think if in the meanwhile we move this functionality on its own branch until is ready?

**Comment by rbarilani** [comment]
@mikiobraun 
Well First, I don't know if such "complex and detailed graph" fit the purpose of the organization page. It seems more interesting to put that in our internal page (in the tech wiki).
Second, as it is, it breaks the layout if you try to re-size the page.
Third, if you try to re-size the page the top navigation is broken because it was simply implemented to handle only two menu entry.

**Comment by mikiobraun** [comment]
The graphs were specifically requested by Jan, so I guess you'd need to discuss with him whether it makes sense.

Concerning the resizing, I had set the flag in the configs and I remember that it also used to work. Maybe something happened there and it's broken? Resizing is a bit complicated because as the raph is based on d3, it is really just a fixed sized svg image. I had tried catching the event and explicitly resizing, but that didn't work.

Maybe there is a way to fix the top navigation?

In any case, I would hate to see these changes go, especially they were the only contribution on the frontend from our onboarding project. Maybe we can both find time to sit together for half an hour next week and try to fix it? I could definitely benefit from some support there.
